packages/tools/create-kadena-app/templates/angular/src/app/app.component.html

```diff
@@ -0,0 +1,36 @@
+<div>
+ <main class="grid">
+   <div class="card">
+     <div class="header">
+       <h1 class="title">Welcome to <span>Kadena!</span></h1>
+     </div>
+     <p>This is the Kadena starter template using nextjs to help you get started on your blockchain development. Use the form below
```

Comment:
@jermaine150 this template mentions Nextjs instead of Angular

Reply:
Thanks, i've updated it.